playbooks/generic-deploy_airship.yml

```diff
@@ -20,12 +20,27 @@
   hosts: ses_nodes[0]
   remote_user: root
   gather_facts: true
+  vars:
+    ansible_python_interpreter: /usr/bin/python2
```

Comment:
what's the problem with python3 here? Anyway, assuming you have the inventory with python3, you can still override it here.

Reply:
this runs on the ses nodes, which are SLE_12, which are python 2.x only

Reply:
I expected this. What I meant is that we don't expect to need to override this on a play basis because something would not work without it set, but more generally about the nodes themselves having no python3. Which point me to the inventory thing.